test/v1/services/CreateAmendUkSavingsAnnualSummaryServiceSpec.scala

```diff
@@ -94,8 +94,16 @@ class CreateAmendUkSavingsAnnualSummaryServiceSpec extends ServiceSpec {
           ("SERVER_ERROR", StandardDownstreamError),
           ("SERVICE_UNAVAILABLE", StandardDownstreamError)
         )
+        val tysErrors = Seq(
+          ("INVALID_TAXYEAR"            -> TaxYearFormatError),
```

Comment:
Needs an underscore because of renaming (so also no need for `.distinct` call now) plus associated change in service itself:
```suggestion
          ("INVALID_TAX_YEAR"            -> TaxYearFormatError),
```